packages/ui/src/auth/useJWT.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+const LOCAL_STORAGE_JWT_KEY = 'jwt';
+
+export function useJWT(): [string | null, (token: string | null) => void] {
+  const item = localStorage.getItem(LOCAL_STORAGE_JWT_KEY);
+
+  const [token, setToken] = useState<null | string>(item);
+
+  return [
+    token,
+    token => {
+      if (token === null) {
+        localStorage.removeItem(LOCAL_STORAGE_JWT_KEY);
```

Comment:
So to end the session, you would do `setToken(null)`?

Reply:
Exactly, which is done in the logout function. Do you have a better idea/pattern in mind?